index.js

```diff
@@ -3,7 +3,7 @@
 const path = require('path');
 const symbols = require('./symbols')
 
-const include_dir = path.relative('.', __dirname, 'include');
+const include_dir = path.relative('.', `${__dirname}/include`);
```

Comment:
Can you elaborate on this change a bit? We don't think this is necessary.

Reply:
`path.relative(from, to)` takes only two prameters. See https://nodejs.org/dist/latest-v16.x/docs/api/path.html#path_path_relative_from_to

Reply:
🤦  i was thinking `path.resolve` Thanks!